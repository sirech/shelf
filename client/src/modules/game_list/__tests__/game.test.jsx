import React from 'react'
import renderer from 'react-test-renderer'

import Game from '../game'

describe('components', () => {
  describe('Game', () => {
    it('renders correctly', () => {
      const game = renderer.create(<Game title='Civ VI' stars={5} />)
      expect(game.toJSON()).toMatchSnapshot()
    })
  })
})
