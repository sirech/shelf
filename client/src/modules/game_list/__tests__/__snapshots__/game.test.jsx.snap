exports[`components Game renders correctly 1`] = `
<li
  className="undefined">
  <h5>
    Civ VI
  </h5>
  <span
    className="undefined">
    <i
      className="fa fa-star"
      onClick={[Function]} />
    <i
      className="fa fa-star"
      onClick={[Function]} />
    <i
      className="fa fa-star"
      onClick={[Function]} />
    <i
      className="fa fa-star"
      onClick={[Function]} />
    <i
      className="fa fa-star"
      onClick={[Function]} />
  </span>
</li>
`;
